operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,236 @@
+# Zuul users guide
+
+## Prerequisites
+
+1. Repository is known by [SCS zuul](https://zuul.scs.community)
+2. Basic ansible knowledge
+3. Basic yaml knowledge
+4. zuul-client installed
+
+Check [SCS zuul projects](https://zuul.scs.community/t/SCS/projects) for your repository to
+be available. If it is missing you need an administrator to get your repository
+configured to zuul.
+
+## Who is it for?
+
+You may have heard about zuul and may ask yourself if it is capable to support you.
+Basically everything you use ansible for can be done using zuul. That is not always
+a good thing since you may get careless and your workload will exceed the CI/CD concept.
+
+If you find yourself doing things under the following list you are at the right place.
+
+1. Code testing
+2. Deployment tests using IaC
+
+If you want to, let's say, monitor something using zuul, that is possible but not the
+intended use case.
+
+## Where do I start?
+
+Right in your projects repository! The only prerequisite you need to fulfill, is that
+your repository you want zuul to work on is known by zuul. This is done by the zuuls
+tenant configuration. To update this configuration you need access to the zuul instance
+or ask an administrator for help.
+
+We consider that zuul knows about your repository so we can get started. There are three
+topics that you should know about. To get jobs running you need the "job" itself. Jobs run
+within "pipeline". The third important thing is to provide a "project" definition.
+
+## Where to save the zuul relavant data?
+
+Zuul will parse all branches of the untrusted repositories that zuul knows about.
```

Comment:
Since we didn't set it explicitly the tenants default value is `false`.
This value is inherited by the projects configuration.

To full fill your request we should update the zuul tenant/projects configuration. Is this correct?